go/store/val/codec.go

```diff
@@ -55,27 +61,26 @@ type Encoding uint8
 
 // Constant Size Encodings
 const (
-	NullEnc    Encoding = 0
-	Int8Enc    Encoding = 1
-	Uint8Enc   Encoding = 2
-	Int16Enc   Encoding = 3
-	Uint16Enc  Encoding = 4
-	Int24Enc   Encoding = 5
-	Uint24Enc  Encoding = 6
+	NullEnc   Encoding = 0
+	Int8Enc   Encoding = 1
+	Uint8Enc  Encoding = 2
+	Int16Enc  Encoding = 3
+	Uint16Enc Encoding = 4
+	// Int24Enc   Encoding = 5
```

Comment:
?

Reply:
still undecided if we're going to support a 3 byte integer encoding